content/applications/finance/accounting/fiscal_localizations/localizations/italy.rst

```diff
@@ -2,78 +2,107 @@
+Agenzia delle Entrate authorities.
+
+SDICoop
+-------
 
 To receive invoices and notifications from third parties, you need to inform the FatturaPA service
-that Odoo is the allowed party to process files for you. To do so, you must setup Odoo's *Codice
-Destinatario* on the FatturaPA portal. The *Codice Destinatario* is ``K95IV18``.
+that Odoo is the allowed party to process files for you. To do so, you must set up Odoo's **Codice
+Destinatario** on the FatturaPA portal.
 
 #. Go to https://ivaservizi.agenziaentrate.gov.it/portale/ and authenticate.
 #. Go to section :menuselection:`Fatture e Corrispettivi`.
-#. Set the user as Legal Party for the VAT number you wish to configure the electronic adress.
+#. Set the user as Legal Party for the VAT number you wish to configure the electronic address.
 #. In :menuselection:`Servizi Disponibili --> Fatturazione Elettronica --> Registrazione
-   dell’indirizzo telematico dove ricevere tutte le fatture elettroniche`, input Odoo's Codice
-   Destinatario (**K95IV18**), then confirm.
+   dell’indirizzo telematico dove ricevere tutte le fatture elettroniche`, insert Odoo's Codice
+   Destinatario (`K95IV18`), then confirm.
 
 Give Odoo permission to process files
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+-------------------------------------
 
 Since the files are transmitted through Odoo's server before being sent to SDICoop or received by
 your database, you need to authorize Odoo to process your files from your database.
 
-To do this, go to :menuselection:`Accounting --> Settings --> Electronic Invoicing` and click on
-*Register*.
-
-.. image:: italy/register.png
-   :align: center
-   :alt: Register button to allow Odoo to process invoices.
+To do so, go to :menuselection:`Accounting --> Configuration --> Settings --> Electronic
+Document Invoicing`. In the :guilabel:`Electronic Document Invoicing`, you can see three different
+modes:
+- Demo
+- Test (experimental)
+- Official (production)
+
+Choosing the right mode
+-----------------------
+- :guilabel:`Demo` mode in Odoo is a simulation environment where all invoices created are **not**
+  automatically sent to the government. In such mode, the invoices need to be manually downloaded
+  as xml files and you then have to upload them on the Agenzia dell'Entrate website.
+- :guilabel:`Test` mode sends invoices to a non-production service.
+- :guilabel:`Official` is a production mode that sends your invoices directly to the Agenzia
+  dell'Entrate.
 
-.. Note::
-   All your files are encrypted upon reception in such a way that only you are able to decrypt them.
-
-Enable FatturaPA on the Sales journal
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Open your sales journal. Under the *Advanced Settings* tab, in the **Electronic invoicing** field,
-check *FatturaPA* and *Save*.
+.. warning::
+   Once in :guilabel:`Official` mode it is not possible to get back to the :guilabel:`Test` and
```

Comment:
Care: you also cannot go from test to official!
We only suggest the "test" mode only for test databases. 

A client creates a test database just to test out the feature, then on its production database he goes straight from demo to production.

Furthermore, the "test" mode only enables you to send invoices to a selected few "fake" destination "CodiceDestinatario": 
```
0803HR0
N8MIMM9
X9XX79Z
```

That's it as of now. In the future we may implement change from test to official.